### PR TITLE
#3540 Offer burned nfts

### LINKED ISSUE #3540: Offer burned nfts
@yangwao I see that you've added this reuqest
> test if it's showing offers on burned nfts where offers did not expire

did you mean to add some column which will track whether NFT is burned or not? Cause the query right now, is fetching all of the NFTs regardless of their state (so you can see the burned ones in the table)

_Originally posted by @petersopko in https://github.com/kodadot/nft-gallery/issues/3477#issuecomment-1192412385_

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

### What's new?

- [x] PR closes #3540
- [ ] Requires deployment <rubick/magick_issue>
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screen Shot 2022-07-26 at 6 03 21 PM

### CODE REVIEW

**Review by roiLeo — COMMENTED** [review]
why do we need to show burned content?
If it's burned, the offer shouldn't exist anymore

**Comment by yangwao** [comment]
> why do we need to show burned content?
> If it's burned, the offer shouldn't exist anymore

the issue with bsx implementation is that if I'm not mistaken, either if the offer is expired or on burned nft, your stake is not being unlocked. 

thus maybe #3401 would be a more excellent way to go, but this way to have it here till then is also good, maybe later removed. As currently there is no other way to see your offers on burned nfts tho.

**Comment by petersopko** [comment]
@prachi00 I'm having some trouble testing this, but as @yangwao mentioned, we're going to move forward with this as (at least) temporary solution, so please:
1. let's merge latest main, balance input was redesigned (currently I can't place offers and therefore cannot test for burned NFTs with my offers)
2. resolve conflicts pls

thank youu 🚀

**Comment by prachi00** [comment]
> @prachi00 I'm having some trouble testing this, but as @yangwao mentioned, we're going to move forward with this as (at least) temporary solution, so please:
> 
> 1. let's merge latest main, balance input was redesigned (currently I can't place offers and therefore cannot test for burned NFTs with my offers)
> 2. resolve conflicts pls
> 
> thank youu 🚀

donee

**Review by petersopko — APPROVED** [review]
wfm

**Comment by petersopko** [comment]
@Jarsen136 hey, since @roiLeo is at holiday, can you check code in this one please? :)

**Review by Jarsen136 — APPROVED** [review]
code lgtm

File: components/bsx/Offer/OffersUserTable.vue [hunk 1]
```diff
@@ -77,7 +84,7 @@
 
 <script lang="ts">
 import { Attribute, emptyArray } from '@kodadot1/minimark'
-import { Component, mixins, Prop } from 'nuxt-property-decorator'
+import { Component, Emit, mixins, Prop, Watch } from 'nuxt-property-decorator'
```
Comment (Jarsen136) [inline components/bsx/Offer/OffersUserTable.vue:87]: `Watch` is nerve been used

**Comment by prachi00** [comment]
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 62.03 USD/KSM ~ 0.806 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0x562377dd5ebfcde7fc3aed7f2027d3de8b0c93fa2862781edc62396660de5804](https://kusama.subscan.io/extrinsic/0x562377dd5ebfcde7fc3aed7f2027d3de8b0c93fa2862781edc62396660de5804)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by petersopko** [comment]
@prachi00 after you've merged and closed this one, I see that the deploy didn't happen, was every conflict resolved etc?

I'm currently facing an issue, where I cannot withdraw the offer from burned the NFT (filtering works, action button doesn't) I saw that it was working for me before, but perhaps, there was something wrong ? Can you check please and reopen if there's some adjustment needed? 

image

**Comment by prachi00** [comment]
> @prachi00 after you've merged and closed this one, I see that the deploy didn't happen, was every conflict resolved etc?
> 
> I'm currently facing an issue, where I cannot withdraw the offer from burned the NFT (filtering works, action button doesn't) I saw that it was working for me before, but perhaps, there was something wrong ? Can you check please and reopen if there's some adjustment needed?
> 
> image

I'll check

**Comment by prachi00** [comment]
> @prachi00 after you've merged and closed this one, I see that the deploy didn't happen, was every conflict resolved etc?
> 
> I'm currently facing an issue, where I cannot withdraw the offer from burned the NFT (filtering works, action button doesn't) I saw that it was working for me before, but perhaps, there was something wrong ? Can you check please and reopen if there's some adjustment needed?
> 
> image

@petersopko I dont think its caused by this but we can revert it and check if it works fine then for you?

**Comment by petersopko** [comment]
@prachi00 you're right, this is not problem of this PR

**Comment by petersopko** [comment]
https://github.com/galacticcouncil/Basilisk-node/issues/543

**Pull request merged**